net/samba4/Config.in

```diff
@@ -0,0 +1,111 @@
+if PACKAGE_samba4
+
+config SAMBA4_SERVER
+	bool "Server (fileserver)"
+	select BUSYBOX_CONFIG_RENICE
+	select BUSYBOX_DEFAULT_RENICE
+	help
+		installs: smbd (daemon) smbpasswd pdbedit testparm smbstatus smbtree
+		This provides the basic fileserver service and is the minimum needed to serve file-shares.
+		good guide: https://fitzcarraldoblog.wordpress.com/2016/10/17/a-correct-method-of-configuring-samba-for-browsing-smb-shares-in-a-home-network/
+	default y
+	
+	config SAMBA4_SERVER_NETBIOS
+		bool "enable Netbios support"
+		depends on SAMBA4_SERVER
+		help
+			installs: nmbd (daemon) nmblookup
+			Builds the nmbd to announce samba resources via netbios.
+			NOTE: Windows 10 (1709+) will not display workgroup netbios resources anymore, unless the old SMB1 feature is reinstalled (not recommend).
+			Therefor this feature can be safely disabled, network shares can still be directly accessed via network paths '\\hostname\sharename' and mounted via 'map network drive'.
+		default y	
+	
+	config SAMBA4_SERVER_VFS
+		bool "enable common VFS modules"
+		depends on SAMBA4_SERVER
+		help
+			provides commonly used vfs modules
+			installs: 
+				modules: vfs_btrfs vfs_fruit vfs_shadow_copy vfs_shadow_copy2 vfs_recycle vfs_fake_perms vfs_readonly vfs_cap vfs_offline vfs_crossrename
+		default y
+	
+	config SAMBA4_SERVER_VFSX
+		bool "enable extendet VFS modules"
+		depends on SAMBA4_SERVER
+		select PACKAGE_libdbus
+		help
+			provides extra vfs modules
+			installs: 
+				modules: vfs_virusfilter vfs_shell_snap vfs_snapper vfs_commit vfs_worm vfs_xattr_tdb vfs_streams_xattr vfs_aio_fork vfs_aio_pthread vfs_linux_xfs_sgid vfs_netatalk vfs_dirsort vfs_fileid
```

Comment:
vfs_snapper --> Requires snapper (not in repo)
vfs_aio_* --> Not sure we needs those at all as we have optional aio support?
vfs_linux_xfs_sgid --> should depend on kmod-fs-xfs?